central/graphql/generator/codegen/case.go

```diff
@@ -0,0 +1,60 @@
+package codegen
+
+// From: https://github.com/golang/protobuf/blob/v1.5.4/protoc-gen-go/generator/generator.go#L2632-L2687
+
+// Is c an ASCII lower-case letter?
+func isASCIILower(c byte) bool {
+	return 'a' <= c && c <= 'z'
+}
+
+// Is c an ASCII digit?
+func isASCIIDigit(c byte) bool {
+	return '0' <= c && c <= '9'
+}
+
+// camelCase returns the CamelCased name.
+// If there is an interior underscore followed by a lower case letter,
+// drop the underscore and convert the letter to upper case.
+// There is a remote possibility of this rewrite causing a name collision,
+// but it's so remote we're prepared to pretend it's nonexistent - since the
+// C++ generator lowercases names, it's extremely unlikely to have two fields
+// with different capitalizations.
+// In short, _my_field_name_2 becomes XMyFieldName_2.
+func camelCase(s string) string {
```

Comment:
Would it be possible (sufficient) for us to use `upperCamelCase` implementation from `tools/generate-helpers/pg-table-bindings/funcs.go` -> maybe we can move that function to some utils and use it. Would that work?

I'm not sure how much names prefixed with `_` are important for us. There is also implementation in https://github.com/protocolbuffers/protobuf-go/blob/b36f8e04bd2e5efbd0bd3df4c450b101beb7de41/internal/strs/strings.go#L92

Reply:
We also have https://github.com/stackrox/stackrox/blob/d423fbb04c1dddac4133b8876b931cb7d8542c27/pkg/postgres/walker/stringutils.go#L31

Reply:
Since this will touch multiple packages I created a separated task for it https://issues.redhat.com/browse/ROX-25712